### ISSUE TITLE
Wappalyzer gives different results for the same website while running multiple times

### ISSUE THREAD

**Issue by Rajasoundarya** [AliasIO/Wappalyzer#1792]
wappalyzer.analyze('https://www.wired.com/2015/11/building-the-star-wars-universe/')
.then(json => {
	var resp=json;
        console.log(resp);
});

if i run this file in command prompt multiple times,everytime the listed technologies vary.its varying from the result of web extension also.
its is taking more time to run this command and sometimes its not ending at all.
How to fix this issue?

the varied responses i got are as follows:
````[ { name: 'Apache',
    confidence: '100',
    version: '',
    icon: 'Apache.svg',
    website: 'http://apache.org',
    categories: [ [Object] ] },
  { name: 'Disqus',
    confidence: '100',
    version: '',
    icon: 'Disqus.svg',
    website: 'http://disqus.com',
    categories: [ [Object] ] },
  { name: 'Fastly',
    confidence: '100',
    version: '',
    icon: 'Fastly.svg',
    website: 'https://www.fastly.com',
    categories: [ [Object] ] },
  { name: 'Typekit',
    confidence: '100',
    version: '',
    icon: 'Typekit.png',
    website: 'http://typekit.com',
    categories: [ [Object] ] },
  { name: 'Varnish',
    confidence: '100',
    version: '',
    icon: 'Varnish.svg',
    website: 'http://www.varnish-cache.org',
    categories: [ [Object] ] },
  { name: 'WordPress',
    confidence: '100',
    version: '',
    icon: 'WordPress.svg',
    website: 'http://wordpress.org',
    categories: [ [Object], [Object] ] },
  { name: 'comScore',
    confidence: '100',
    version: '',
    icon: 'comScore.png',
    website: 'http://comscore.com',
    categories: [ [Object] ] },
  { name: 'PHP',
    confidence: '0',
    version: '',
    icon: 'PHP.svg',
    website: 'http://php.net',
    categories: [ [Object] ] } ]````
````[ { name: 'Apache',
    confidence: '100',
    version: '',
    icon: 'Apache.svg',
    website: 'http://apache.org',
    categories: [ [Object] ] },
  { name: 'Disqus',
    confidence: '100',
    version: '',
    icon: 'Disqus.svg',
    website: 'http://disqus.com',
    categories: [ [Object] ] },
  { name: 'DoubleClick for Publishers (DFP)',
    confidence: '100',
    version: '',
    icon: 'DoubleClick.svg',
    website: 'http://www.google.com/dfp',
    categories: [ [Object] ] },
  { name: 'Fastly',
    confidence: '100',
    version: '',
    icon: 'Fastly.svg',
    website: 'https://www.fastly.com',
    categories: [ [Object] ] },
  { name: 'Google Tag Manager',
    confidence: '100',
    version: '',
    icon: 'Google Tag Manager.png',
    website: 'http://www.google.com/tagmanager',
    categories: [ [Object] ] },
  { name: 'Hammer.js',
    confidence: '100',
    version: '',
    icon: 'Hammer.js.png',
    website: 'http://hammerjs.github.io',
    categories: [ [Object] ] },
  { name: 'Parse.ly',
    confidence: '100',
    version: '',
    icon: 'Parse.ly.png',
    website: 'http://parse.ly',
    categories: [ [Object] ] },
  { name: 'SiteCatalyst',
    confidence: '100',
    version: '',
    icon: 'SiteCatalyst.png',
    website: 'http://www.adobe.com/solutions/digital-marketing.html',
    categories: [ [Object] ] },
  { name: 'Typekit',
    confidence: '100',
    version: '',
    icon: 'Typekit.png',
    website: 'http://typekit.com',
    categories: [ [Object] ] },
  { name: 'Varnish',
    confidence: '100',
    version: '',
    icon: 'Varnish.svg',
    website: 'http://www.varnish-cache.org',
    categories: [ [Object] ] },
  { name: 'WordPress',
    confidence: '100',
    version: '',
    icon: 'WordPress.svg',
    website: 'http://wordpress.org',
    categories: [ [Object], [Object] ] },
  { name: 'comScore',
    confidence: '100',
    version: '',
    icon: 'comScore.png',
    website: 'http://comscore.com',
    categories: [ [Object] ] },
  { name: 'jQuery',
    confidence: '100',
    version: '',
    icon: 'jQuery.svg',
    website: 'http://jquery.com',
    categories: [ [Object] ] },
  { name: 'PHP',
    confidence: '0',
    version: '',
    icon: 'PHP.svg',
    website: 'http://php.net',
    categories: [ [Object] ] } ]````

**Comment by jvoisin** [comment]
Can you please show us the different outputs?

(You can surround code blocks with triple backticks (```) to make them more readable)

**Comment by anthonycali** [comment]
Yeah I have been having similar issues. Mentioned [here](https://github.com/AliasIO/Wappalyzer/issues/1789)

**Comment by AlexByte** [comment]
Give outputs foo this site

**Comment by AliasIO** [comment]
This should be fixed as per #1789.
